test/e2e/flowaggregator_test.go

```diff
@@ -574,6 +587,26 @@ func checkRecordsForFlows(t *testing.T, data *TestData, srcIP string, dstIP stri
 	assert.GreaterOrEqualf(t, dataRecordsCount, expectedNumDataRecords, "IPFIX collector should receive expected number of flow records. Considered records: ", len(recordSlices))
 }
 
+func checkRecordsForToExternalFlows(t *testing.T, data *TestData, srcPodName string, srcIP string, dstIP string) {
+	timeStart := time.Now()
+	cmdStr := fmt.Sprintf("curl %s", externalServer)
+	data.runCommandFromPod(testNamespace, srcPodName, nginxContainerName, []string{"/bin/sh", "-c", cmdStr})
+	collectorOutput := getCollectorOutput(t, srcIP, dstIP, timeStart)
+
+	recordSlices := getRecordsFromOutput(collectorOutput)
+	for _, record := range recordSlices {
+		if strings.Contains(record, srcIP) && strings.Contains(record, dstIP) {
+			checkPodAndNodeData(t, record, srcPodName, controlPlaneNodeName(), "", "")
+			checkFlowType(t, record, ipfixregistry.FlowTypeToExternal)
+			// skipping byte count checks in Kind cluster,
```

Comment:
Same here.. start with caps and make better use of 80chars in a line.

"Since the OVS userspace conntrack implementation doesn't maintain packet or byte counter statistics, skip the check for Kind clusters."